scripts/core/superdesk-api.d.ts

```diff
@@ -3717,5 +3717,9 @@ declare module 'superdesk-api' {
         /** original source of the data, eg. wikidata */
         original_source?: string;
         parent?: string;
+
+        // AI generated subjects
+        creator?: string;
```

Comment:
let's overwrite types in the extension instead

![image](https://github.com/user-attachments/assets/a88bd46a-49ec-4f61-8bbe-c2ee57675a93)